package.json

```diff
@@ -50,11 +52,14 @@
     "pure-rand": "^1.6.2"
   },
   "devDependencies": {
+    "@microsoft/api-extractor": "^7.0.39",
     "@types/jest": "^24.0.11",
     "@types/node": "^11.11.3",
     "benchmark": "^2.1.4",
     "browserify": "^16.2.3",
     "coveralls": "^3.0.3",
+    "docsify-tools": "^1.0.8",
+    "gh-pages": "^2.0.1",
```

Comment:
No need for the gh-pages package for the moment. I already handle the deployment of documentation in https://github.com/dubzzz/fast-check/blob/master/.travis.yml#L63:L76